site2/docs/reference-cli-tools.md

```diff
@@ -437,6 +437,7 @@ Options
 |`-v`, `--encryption-key-value-file`|The file which contains the private key to decrypt payload||
 |`-h`, `--help`|Help message|false|
 |`--conf-file`|Configuration file||
+|`-m`, `--num-messages`|Number of messages to consume in total. If <= 0, it will keep consuming.|0|
```

Comment:
```suggestion
|`-m`, `--num-messages`|Number of messages to consume in total. If the value is equal to or smaller than 0, it keeps consuming messages.|0|
```

Reply:
Please keep consistent in other descriptions.

Reply:
done